README.md

```diff
@@ -20,18 +20,20 @@ If you don't have a Snowflake account yet, you can [sign up for a 30-day free tr
 You can use [miniconda][miniconda], [anaconda][anaconda], or [virtualenv][virtualenv]
 to create a Python 3.8, 3.9, 3.10 or 3.11 virtual environment.
 
+For Snowpark pandas, only Python 3.9, 3.10, or 3.11 is supported.
+
 To have the best experience when using it with UDFs, [creating a local conda environment with the Snowflake channel][use snowflake channel] is recommended.
 
 ### Install the library to the Python virtual environment
 ```bash
 pip install snowflake-snowpark-python
 ```
-Optionally, you need to install pandas in the same environment if you want to use pandas-related features:
```

Comment:
we should keep this. There is possible that customer only use native pandas not modin.

Reply:
I don't see any downside to having the Modin target on the README (the Modin package isn't that large). Since the Modin target also installs native pandas, users can use both native pandas and Snowpark pandas if they want to.

Reply:
This is specifically called out by "Optionally, you need to install pandas and Modin in the same environment if you want to use Snowpark pandas features:"

Reply:
sounds like a question to the python API team. cc @sfc-gh-jdu

Reply:
I think we should have lines for both `snowflake-snowpark-python[pandas]` and `snowflake-snowpark-python[modin]` as separate (one for snowpark + native pandas, the other for snowpark pandas).

Reply:
Ok let me make that change.